### PR TITLE
fixes for 7981 and 7979

### PR DESCRIPTION
**Pull request by RemcoSimonides**
#7981
- [x] When trying to validate a picture with the dedicated button, clicking close to the border of the button activate the 'click effect' but without the 'validate action'. It's the same with the other picture related buttons (like delete). The button's hitbox should be the circle and not just the icon inside, the user could think the app isn't working properly :
Validating picture

#7979
- [x] (A) Overview of org members is displayed
- [x] (C) Style CSS - Organization ressource

### CODE REVIEW

File: apps/crm/crm/src/app/crm/pages/organization/organization.component.ts [hunk 1]
```diff
@@ -97,15 +97,20 @@ export class OrganizationComponent implements OnInit {
...
-    const members = await this.organizationService.getMembers(this.orgId);
-    return members.map((m) => ({
-      ...m,
-      userId: m.uid,
-      edit: {
-        id: m.uid,
-        link: `/c/o/dashboard/crm/user/${m.uid}`,
-      },
-    }));
+    const [ members, role ] = await Promise.all([
```
Comment (phpgeekfr) [inline apps/crm/crm/src/app/crm/pages/organization/organization.component.ts:100]: bad formatting ;)

File: apps/crm/crm/src/app/crm/pages/organization/organization.component.ts [hunk 2]
```diff
@@ -97,15 +97,20 @@ export class OrganizationComponent implements OnInit {
...
+    return members.map((m) => {
+      return {
+        ...createOrganizationMember(m, role.roles[m.uid] ? role.roles[m.uid] : undefined),
+        userId: m.uid,
+        edit: {
+          id: m.uid,
+          link: `/c/o/dashboard/crm/user/${m.uid}`,
+        },
+      };
+    });
```
Comment (phpgeekfr) [inline apps/crm/crm/src/app/crm/pages/organization/organization.component.ts:113]:
```suggestion
    return members.map((m) => ({
        ...createOrganizationMember(m, role.roles[m.uid] ? role.roles[m.uid] : undefined),
        userId: m.uid,
        edit: {
          id: m.uid,
          link: `/c/o/dashboard/crm/user/${m.uid}`,
        },
      });
```

File: libs/organization/src/lib/organization/+state/organization.service.ts [hunk 3]
```diff
@@ -158,14 +158,11 @@ export class OrganizationService extends CollectionService<OrganizationState> {
     return this.update(orgId, { userIds });
   }
 
-  public async getMembers(orgId: string): Promise<OrganizationMember[]> {
+  public async getMembers(orgId: string): Promise<PublicUser[]> {
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/+state/organization.service.ts:161]: all parts of the code that uses `getMembers(` does not need the role attribute anymore ?

**Review by phpgeekfr — APPROVED** [review]

File: libs/organization/src/lib/organization/+state/organization.service.ts (hunk 3 above)
Comment (RemcoSimonides) [reply libs/organization/src/lib/organization/+state/organization.service.ts:161]: That's right

**Pull request merged**